### PR TITLE
Bug 1989722: Add SoftTopologyAndDuplicates profile

### PR DESCRIPTION
**Pull request by damemi**
Adds a copy of the `TopologyAndDuplicates` profile which includes soft constraints

Ref https://issues.redhat.com/browse/WRKLDS-302

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/cc @ingvagabund @kasturinarra

File: manifests/4.9/kube-descheduler-operator.crd.yaml [hunk 1]
```diff
@@ -92,6 +92,9 @@ spec:
                   - TopologyAndDuplicates
                   - LifecycleAndUtilization
                   - DevPreviewLongLifecycle
+                  - SoftTopologyAndDuplicates
+                  type: string
```
Comment (ingvagabund) [inline manifests/4.9/kube-descheduler-operator.crd.yaml:96]: duplicated line

File: manifests/4.9/kube-descheduler-operator.crd.yaml [hunk 2]
```diff
@@ -92,6 +92,9 @@ spec:
...
+                type: array
```
Comment (ingvagabund) [inline manifests/4.9/kube-descheduler-operator.crd.yaml:97]: duplicated line

File: manifests/4.9/kube-descheduler-operator.crd.yaml (hunk 1 above)
Comment (damemi) [reply manifests/4.9/kube-descheduler-operator.crd.yaml:96]: Sorry, which line is this a duplicate of?

Comment (damemi) [reply manifests/4.9/kube-descheduler-operator.crd.yaml:96]: Ah I see, it's above on L89... weird, this was generated

Comment (ingvagabund) [reply manifests/4.9/kube-descheduler-operator.crd.yaml:96]: Gotcha

**Pull request merged**